src/core/core.element.js

```diff
@@ -6,7 +6,9 @@ import {isNumber} from '../helpers/helpers.math';
 class Element {
 
 	constructor(configuration) {
-		extend(this, configuration);
+		if (configuration) {
```

Comment:
This is certainly an improvement, but I wonder if we even need to call `extend` in the base class. Can we just do it in the couple places that need it? (I think maybe tooltip and scale). The "actual" elements (i.e. arc, point, line, etc.) only use this in the tests and not the real code

Reply:
At least Scales (extending Element) are constructed with bunch of properties.

Reply:
Yeah, not critical. Just a thought. I'm fine with this the way it is